furnace.c

```diff
@@ -63,8 +63,17 @@ typedef struct {
   uint8_t         pulse_count;
   absolute_time_t magnetron_deadline;
 #endif
+  
+#if CONFIG_WATER
+  uint8_t pwm_water;
+#endif
 } furnace_context_t;
 
+#if CONFIG_WATER
+  #define WATER_PIN 10
```

Comment:
Not saying that It's wrong, but is there any reason to pick GPIO 10?

Reply:
It is right across the other pwm pin, it makes it easier to remember, and also uses other pwm slice, which allows us to have different prescaler.

Reply:
actually, it is already taken by magnetron, i changed it to GPIO12